robot_smach_states/src/robot_smach_states/world_model.py

```diff
@@ -220,6 +254,33 @@ def __init__(self, robot, entityDes, objectIDsDes=None, searchArea="on_top_of",
                                    transitions={'done': 'done'})
 
 
+class CheckVolumeEmpty(smach.StateMachine):
+    def __init__(self, robot, entity_des, volume="on_top_of", volume_threshold=0.0):
+        """ Constructor
+
+        :param robot: robot object
+        :param entity_des: EdEntityDesignator indicating the (furniture) object to check
+        :param volume: string defining of the entity to be checked, default = on_top_of
```

Comment:
unclear sentence, missing "part"?

Reply:
changed to 'volume of the entity to be checked'